var/spack/repos/builtin/packages/r-loo/package.py

```diff
@@ -11,8 +11,8 @@ class RLoo(RPackage):
        BayesianModels"""
 
     homepage = "https://mc-stan.org/loo"
-    url      = "https://cran.r-project.org/src/contrib/loo_2.1.0.tar.gz"
-    list_url = "https://cran.r-project.org/src/contrib/Archive/loo"
+    url      = "https://cloud.r-project.org/src/contrib/loo_2.1.0.tar.gz"
+    list_url = "https://cloud.r-project.org/src/contrib/Archive/loo"
 
     version('2.1.0', sha256='1bf4a1ef85d151577ff96d4cf2a29c9ef24370b0b1eb08c70dcf45884350e87d')
 
```

Comment:
> SystemRequirements:	pandoc (>= 1.12.3), pandoc-citeproc

Reply:
Looking at the PR for the original package, #11448, it was determined that pandoc was not really a dependency for r-loo.

Reply:
I went ahead and added the dependency for pandoc.